tendrl/ceph_integration/objects/rbd/atoms/create/__init__.py

```diff
@@ -1,18 +1,166 @@
+import json
+import uuid
+
 from tendrl.ceph_integration.manager.crud import Crud
 from tendrl.ceph_integration.manager.exceptions import \
     RequestStateError
 from tendrl.commons import objects
+from tendrl.ceph_integration.objects.pool import Pool
 from tendrl.ceph_integration.objects.rbd import Rbd
 from tendrl.commons.event import Event
+from tendrl.commons.objects.job import Job
 from tendrl.commons.message import Message
 
 
 class Create(objects.BaseAtom):
     obj = Rbd
```

Comment:
You dont have to set the object for an atom implicitly, It is done by the commons framework while creating that specific namespace